frontend/src/FullPageRoutes/Profile.tsx

```diff
@@ -19,64 +32,80 @@ const useStyles = makeStyles({
   },
 })
 
-type UserProfile = {
-  first_name: string
-  last_name: string
-  phone_number?: string
-  age?: number
-  gender: string
-}
-
 export default function Profile(): JSX.Element {
   const classes = useStyles()
 
-  const [username, setUsername] = React.useState<string>("")
+  const dummyUserProfile: UserProfile = {
+    username: "nope",
+    data: {
+      first_name: "nope",
+      last_name: "nope",
+      phone_number: "nope",
+      age: -1,
+      gender: "nope",
+    },
+  }
```

Comment:
It seems odd to create what is essentially default values, then immediately attempt to overwrite them. If the request fails, the user will see "nope" everywhere, which is probably not what we want.

Reply:
this was just for testing